packages/lodestar-utils/src/json.ts

```diff
@@ -7,27 +7,31 @@ export function errorToObject(obj: Error): Json {
 }
 
 export function toJson(arg: unknown): Json {
-  switch (typeof arg) {
-    case "bigint":
-    case "symbol":
-    case "function":
-      return arg.toString();
+  try {
+    switch (typeof arg) {
+      case "bigint":
+      case "symbol":
+      case "function":
+        return arg.toString();
 
-    case "object":
-      if (arg === null) return "null";
-      if (Array.isArray(arg)) return arg.map(toJson);
-      if (arg instanceof Uint8Array) return toHexString(arg);
-      if (arg instanceof LodestarError) return toJson(arg.toObject());
-      if (arg instanceof Error) return toJson(errorToObject(arg));
-      return mapValues(arg, (value) => toJson(value)) as Json;
+      case "object":
+        if (arg === null) return "null";
+        if (Array.isArray(arg)) return arg.map(toJson);
+        if (arg instanceof Uint8Array) return toHexString(arg);
+        if (arg instanceof LodestarError) return toJson(arg.toObject());
+        if (arg instanceof Error) return arg.message;
```

Comment:
We should include both the message and stack. Fixes for this issue are included in this PR https://github.com/ChainSafe/lodestar/pull/1669 too